src/util/util.ts

```diff
@@ -49,15 +49,13 @@ export function checkPropsUndefined(props: object): boolean {
 }
 
 // see https://stackoverflow.com/a/38552302
-export function parseJwt(token: string) {
+export function parseJwt(token: string): any {
```

Comment:
Is it possible to add a type or a comment why `any` is ok in this case?

Reply:
Fixed in https://github.com/cstenglein/raspiblitz-web/pull/155 :)